src/components/AuthForm.js

```diff
@@ -313,8 +314,9 @@ export default class AuthForm extends Component {
   onChange = ({formData}: {formData: Object}) => {
     const {authType} = formData;
     const {schema, uiSchema} = authSchemas[authType];
-    const specificFormData = /fxa|anonymous/.test(authType) ? omit(formData, ["credentials"])
-                                                            : {credentials: {}, ...formData};
+    const specificFormData = ["anonymous", "fxa", "portier"].includes(authType)
+      ? omit(formData, ["credentials"])
+      : {credentials: {}, ...formData};
```

Comment:
Yeah we forgot to clean credentials when handling Portier auth form data in #395.